lesson02/assignment/css/style.scss

```diff
@@ -3,4 +3,134 @@
 @import 'settings.variables';
 @import 'settings.responsive';
 @import 'components.content';
```

Comment:
I don't see any new partials created and used. These default imports are for variables. I'd like to see you take advantage of partials for this assignment.